ACE/ace/SOCK_Dgram.cpp

```diff
@@ -339,7 +345,7 @@ ACE_SOCK_Dgram::recv (iovec iov[],
 #endif /* ACE_HAS_SOCKADDR_MSG_NAME */
   recv_msg.msg_namelen = addr.get_size ();
 
-#if defined (ACE_HAS_4_4BSD_SENDMSG_RECVMSG)
+#ifdef ACE_USE_MSG_CONTROL
```

Comment:
Closing comment is no longer correct.

Reply:
Good reason not to use comments for things that the editor/IDE should take care of.